src/Command/InstallRecipesCommand.php

```diff
@@ -36,15 +36,20 @@ public function __construct(/* cannot be type-hinted */ $flex, string $rootDir)
 
     protected function configure()
     {
-        $this->setName('symfony:sync-recipes')
-            ->setAliases(['sync-recipes', 'fix-recipes'])
+        $this->setName('symfony:recipes:install')
+            ->setAliases(['symfony:sync-recipes', 'sync-recipes', 'fix-recipes'])
             ->setDescription('Installs or reinstalls recipes for already installed packages.')
+            // TODO : Separate the force option in an update command
             ->addOption('force', null, InputOption::VALUE_NONE, 'Ignore the "symfony.lock" file and overwrite existing files')
         ;
     }
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
+        if (\in_array($input->getFirstArgument(), $this->getAliases(), true)) {
+            trigger_error(sprintf('"%s" is deprecated. Please use `symfony:recipes:install` instead.', $input->getFirstArgument()), E_USER_DEPRECATED);
```

Comment:
I think we don't need to trigger this: using the alias is fine, and we're not going to release flex v2 just for this.

Reply:
it was requested by @fabpot, but I can delete it, if it's good for both

Reply:
I'm in favor of removing it
/cc @fabpot for confirmation